xdem/coreg/base.py

```diff
@@ -2627,7 +2701,7 @@ def copy(self: CoregType) -> CoregType:
         """Return an identical copy of the class."""
         new_coreg = self.__new__(type(self))
 
-        new_coreg.__dict__ = {key: copy.deepcopy(value) for key, value in self.__dict__.items() if key != "pipeline"}
+        new_coreg.__dict__ = {key: copy.copy(value) for key, value in self.__dict__.items() if key != "pipeline"}
```

Comment:
What does this change do?

Reply:
Forgot to add this in the `CoregPipeline` to mirror an issue fixed in #530 for `Coreg`